nft-client/src/main.rs

```diff
@@ -76,9 +75,11 @@ impl Deserial for NFTContractAddressState {
     }
 }
 
-/// The NFT contract state.
+/// The NFT contract state returned by the view function `view`, which is not
+/// part of CIS2 and is only implemented for debugging purposes.
+///
 /// Important: this structure matches the NFT contract implementations and
-/// cannot be assumed for any CIS1 contract state, as the CIS1 does not restrict
+/// cannot be assumed for any CIS2 contract state, as the CIS2 does not restrict
```

Comment:
```suggestion
/// cannot be assumed to work for any CIS2 contract state, as the CIS2 does not restrict
```
In multiple locations.